Description: sort stopwords in searchtools.js
 The order of stopwords in searchtools.js would be random if hash randomization
 was enabled, breaking dh_sphinxdoc. This patch makes the order deterministic.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: yes, https://bitbucket.org/birkenfeld/sphinx/commits/6cf5320e65
Last-Update: 2012-12-08

--- a/sphinx/search/__init__.py
+++ b/sphinx/search/__init__.py
@@ -283,5 +283,5 @@
     def context_for_searchtool(self):
         return dict(
             search_language_stemming_code = self.lang.js_stemmer_code,
-            search_language_stop_words = jsdump.dumps(self.lang.stopwords),
+            search_language_stop_words = jsdump.dumps(sorted(self.lang.stopwords)),
         )
